tests/libnode/node.go

```diff
@@ -105,6 +90,7 @@ func CleanNodes() {
 
 		if node.Spec.Unschedulable {
 			new.Spec.Unschedulable = false
+			found = true
```

Comment:
So it looks to me like we had in general a side-effect here on the cleanup and the conformance tests would just uncordon all nodes, even ones which we did not specifically cordon for a test. can that be?

Reply:
Apart from that, not using kubectl/oc makes a lot of sense. :+1:

Reply:
We were uncordoning all nodes once via `kubectl` then uncordoning all nodes that had taints / the kubevirt test labels. 

I see where you're getting at ... In order to make this leave user cordoned nodes cordoned, we'd have to look at all tests that are cordoning nodes and assure they are also setting the kubevirt testing label.

Reply:
Yes. I would even be inclined to just say: Hey was broken before, still not fixed but better now. However this intial mistake (that we did not have the kubelet in place) may have prevented some users actually from getting their nodes made schedulable again when running the conformance tests :D

Reply:
I'll remove this line, then investigate the code base cordoning + setting the KubeVirt testing label describe in my previous comment.

Reply:
@rmohr do you think the code is better now ?

Reply:
Hm, from a conformance test perspective it maybe ok now, but do our normal tests depend on uncordoning the nodes this way? Would we cordon as a side-effect more and more nodes in tests?

Reply:
From what I found, all the tests that cordon nodes uncordon them afterward as part of their particular test teardown - meaning this `CleanNodes` uncordoning is really redundant. These are:
- https://github.com/kubevirt/kubevirt/blob/4d9e95e14f414521825be14ed6cdda8e52c04f34/tests/virt_control_plane_test.go#L135
- https://github.com/kubevirt/kubevirt/blob/3a0c874af8d36121f5c98145ea72e1ef27cd37a7/tests/migration_test.go#L2758

... what I can't do right now is assure these are the only instances on node cordoning on the project ... People may use other ways of doing that, rather than use this `libnode` pkg.

Reply:
that is good enough for me. Thanks for checking.